cmd/platform_ui/main.go

```diff
@@ -79,10 +81,19 @@ func main() {
 		schemaLoader = loader.CachedFactory(schemaLoader, cachex)
 	}
 
-	vaultCli, err := providers.NewVaultClient(cfg.KeyStore.Address, cfg.KeyStore.Token)
-	if err != nil {
-		log.Error(ctx, "cannot init vault client: ", "err", err)
-		return
+	var vaultCli *vault.Client
+	if cfg.VaultUserPassAuthEnabled {
+		vaultCli, err = providers.NewVaultClientWithUserPassAuth(ctx, cfg.KeyStore.Address, cfg.VaultUserPassAuthPassword)
+		if err != nil {
+			log.Error(ctx, "cannot init vault client with Kubernetes Auth: ", "err", err)
+			return
+		}
+	} else {
+		vaultCli, err = providers.NewVaultClient(cfg.KeyStore.Address, cfg.KeyStore.Token)
+		if err != nil {
+			log.Error(ctx, "cannot init vault client: ", "err", err)
+			return
+		}
```

Comment:
Same comment as in cmd/issuer_initializer/main.go. I think we can reuse this code